.gitignore

```diff
@@ -39,3 +39,6 @@ setup/win32/static/postgresql*.exe
 /man/
 /share/
 /src/
+
+# avoid adding it after generation from tsconfig command
+tsconfig.json
```

Comment:
Please add the missing new line. This will avoid to break the last line blame when adding new line.